packages/firebase_performance/ios/Classes/FLTFirebasePerformance.m

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FirebasePerformancePlugin.h"
+
+@interface FLTFirebasePerformance ()
+@property FIRPerformance *performance;
+@end
+
+@implementation FLTFirebasePerformance
++ (void)registerWithRegistrar:(nonnull NSObject<FlutterPluginRegistrar> *)registrar {
+}
+
++ (void)sharedInstanceWithCall:(FlutterMethodCall *)call result:(FlutterResult)result {
```

Comment:
From an Objective-C developer perspective, I feel this name could be confusing. Normally in Objective-C `sharedInstance` would be an idempotent static method that returns a singleton instance of the class. Here, you're performing a non-idempotent allocation action and returning nothing. Also you're not doing anything with the result argument. Perhaps another name would make it more clear what's going on.